src/epd/GxEPD2_213_BN.cpp

```diff
@@ -385,16 +381,32 @@ const unsigned char GxEPD2_213_BN::lut_partial[] PROGMEM =
   0x22, 0x22, 0x22, 0x22, 0x22, 0x22, 0x0, 0x0, 0x0,
 };
 
+void GxEPD2_213_BN::_Init_Full()
+{
+  _InitDisplay();
+  _writeCommand(0x3C);  // Border Waveform
+  _writeData(0x05);
+  _writeCommand(0x18); //Read built-in temperature sensor
+  _writeData(0x80);
+  _using_partial_mode = false;
+}
+
+
 void GxEPD2_213_BN::_Init_Part()
 {
+  _InitDisplay();
+  _writeCommand(0x3C);  // Border Waveform
+  _writeData(0xC0);     // Floating (less noise-accumulation?)
```

Comment:
Note border waveform change to 0xC0 - "Hi-Z".
Previous value was 0x05 - "Follow LUT". I believe for the fast-refresh operation this was "VCOM"(?)
This change seems to prevent "static" accumulating on the display border, during fast-refresh.
It would be good to get confirmation that this change is safe and appropriate.